R/Data-methods.R

```diff
@@ -33,12 +33,13 @@ setMethod(
 #'
 #' A method that creates a plot for [`Data`] object.
 #'
-#' @param x (`GeneralData`)\cr object we want to plot.
+#' @param x (`Data`)\cr object we want to plot.
 #' @param y (`missing`)\cr missing object, for compatibility with the generic function.
 #' @param blind (`flag`)\cr indicates whether to blind the data.
 #'   If `TRUE`, then placebo subjects are reported at the same level
 #'   as the active dose level in the corresponding cohort,
 #'   and DLTs are always assigned to the first subjects in a cohort.
+#' @param no_legend (`flag`)\cr should not the legend be added?
```

Comment:
```suggestion
#' @param legend (`flag`)\cr whether the legend should be added.
```
easier to understand with positive wording of this.